.github/workflows/ci.yml

```diff
@@ -74,7 +74,7 @@ jobs:
         working-directory: ./contract-examples
       - name: Install AvalancheGo Release
         shell: bash
-        run: BASEDIR=/tmp/e2e-test AVALANCHEGO_BUILD_PATH=/tmp/e2e-test/avalanchego ./scripts/install_avalanchego_release.sh
+        run: BASE_DIR=/tmp/e2e-test AVALANCHEGO_BUILD_PATH=/tmp/e2e-test/avalanchego ./scripts/install_avalanchego_release.sh
```

Comment:
Can we align all of these `BASEDIR` and `BASE_DIR` to be the same? The previous bug was that they were ever different as the intended behavior is to allow someone to populate the environment variable and use the default if not populated.

Reply:
I don't mind doing that, but it looks like it was done on purpose as the pattern repeats itself in a couple of places, `AVALANCHE_VERSION` vs `AVALANCHE_GO_VERSION` for instance. `BASEDIR` is only meant to be used inside the script.

If I assume that using a "private" variable name like that is justified, I think it would be more pragmatic to check if the user has defined `BASEDIR` instead of `BASE_DIR` and error out with an explanation. I can do the same for `AVALANCHE_GO_VERSION` vs `AVALANCHE_VERSION`

Reply:
In this case, I think both should be public environment variables that can be easily overridden by the user and I'd prefer to be consistent in using the same environment variable in the script that the user needs to override rather than renaming in the background.

I think all of the renaming is confusing when looking at the script and wanting to know how to override a variable or where it comes from.

If we want to go back to having public vs. private env variables, I think we should be explicit and follow the convention of using lowercase for private env variables.